tests/tamr_client/fake.py

```diff
@@ -18,7 +18,27 @@
 fake_json_dir = tests_tc_dir / "fake_json"
 
 
-def _to_kwargs(fake):
+class WrongRequestBody(Exception):
+    """Raised when the body of a request does not match the value expected during
+    testing
+    """
+
+    pass
+
+
+def check_request_body(request, request_body, status, response_json):
+    if isinstance(request_body, list):
+        caught_body = [loads(x.decode("utf-8")) for x in request.body]
+        if caught_body != request_body:
+            raise WrongRequestBody(caught_body)
+    elif request_body is not None:
+        caught_body = loads(request.body.decode("utf-8"))
+        if caught_body != request_body:
```

Comment:
There is an intentional decision here to compare these as JSON objects instead of strings, i.e. to do `loads(request.body.decode("utf-8")` rather than `dumps(request_body)`.  In two strings dumped from two effectively identical JSON dictionaries, a misleading inequality can result from the arbitrary ordering of elements contained in the dictionaries.